### PR TITLE
feat: Add a map to show the locations of the fellows

### LINKED ISSUE #10: Add a map showing all the countries that fellows are located in
Underneath projects and fellows, have an interactive map that displays where everyone is located.

- [ ] When you click on the pin location for that fellow, it gives you a card with information about the fellow (similar to the fellow section

You can chose to have pins or use circular profile pictures to show where everyone is on the map. You could remove the fellows section and replace it with the map.

### PR DESCRIPTION
**Pull request by Shofiya2003**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### What did we use?
- Google maps API to embed the map and locations of the fellows

### How the map looks?
image

Closes #10

### CODE REVIEW

**Comment by sumana2001** [comment]
The map should be placed at the end according to the navbar. Please do that and make sure you add an id or class so that the maps navbar button takes you to the maps part.  @Shofiya2003 @Pravallika-Myneni

**Comment by Shofiya2003** [comment]
> 

okay

**Comment by Shofiya2003** [comment]
@sumana2001 made the changes.

File: _data/fellows.yml [hunk 1]
```diff
@@ -228,3 +228,17 @@
 
     
 
+- name: Sai Pravallika Myneni
```
Comment (sumana2001) [inline _data/fellows.yml:231]: @Pravallika-Myneni 's details are already there in the deployed website. This is repetitive. Please remove it

Comment (Shofiya2003) [reply _data/fellows.yml:222]: removed the repeated details and kept the one with complete info @sumana2001

**Pull request merged**